snitun/client/connector.py

```diff
@@ -51,93 +228,79 @@ async def handler(
         channel: MultiplexerChannel,
     ) -> None:
         """Handle new connection from SNIProxy."""
-        from_endpoint = None
-        from_peer = None
-
-        _LOGGER.debug(
-            "Receive from %s a request for %s",
-            channel.ip_address,
-            self._end_host,
-        )
+        _LOGGER.debug("Receive from %s a request", channel.ip_address)
 
         # Check policy
         if not self._whitelist_policy(channel.ip_address):
             _LOGGER.warning("Block request from %s per policy", channel.ip_address)
             await multiplexer.delete_channel(channel)
             return
 
+        transport = ChannelTransport(self._loop, channel)
+        # The request_handler is the aiohttp RequestHandler
+        # that is generated from the protocol_factory that
+        # was passed in the constructor.
+        request_handler = self._protocol_factory()
+        # Performance: We could avoid the task here if
```

Comment:
This doesn't make much difference in practice. The other PR to switch to a deque has a thought on eliminating the frequent small tasks which is a big source of the performance and latency